README.rst

```diff
@@ -12,8 +12,7 @@ modifications to ensure idempotency.
 Requirements
 ------------
 
-Currently, the role only supports `CentOS`_ and
-`Red Hat Enterprise Linux (RHEL)`_ EL7 distribution flavors.
+Currently, the role only supports `CentOS`_ , `Red Hat Enterprise Linux (RHEL)`_ EL7, `Ubuntu`_ LTS >= 18 distribution flavors.
```

Comment:
Please wrap the line at 79 characters to keep it readable in the editor.

Reply:
Updated.